services/aws_extended_sqs_client/aws_extended_sqs_client.go

```diff
@@ -204,12 +267,14 @@ func (c *AwsExtendedSQSClient) checkMessageAttributes(attributes map[string]*aws
 	return nil
 }
 
-func (c *AwsExtendedSQSClient) isLarge(input *aws_sqs.SendMessageInput) bool {
+func (c *AwsExtendedSQSClient) isLarge(input *aws_sqs.SendMessageInput, logger logrus.FieldLogger) bool {
```

Comment:
can we use logger from client instead of passing as value?

Reply:
would like to preserve the fields set in line 54 for the logger, so we pass it as argument